examples/examples-cpu/nyc-taxi-snowflake/xgboost-dask.ipynb

```diff
@@ -35,19 +35,19 @@
    "outputs": [],
```

Comment:
<p>autoformat here too, I think old way is better</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/saturncloud/examples/pull/60/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>probably a reasonable way around this is to define the kwargs above. Like </p><p> </p><pre class="ql-syntax" spellcheck="false">data = taxi_train[features]
label = taxi_train[y_col]
dtrain = xgb.dask.DaskDMatrix(client=client, data=data, label=label)
</pre><p> </p>

Reply:
<p>the relevant line is 96 characters, so if I just make the variable <code>dtrain_dmatrix</code>,  I think <code>black</code>  will  force a line break and give us back the formatting we want :)</p>

Reply:
<p>ha nope that didn't work. I like Julia's idea.</p>

Reply:
I did this in https://github.com/saturncloud/examples/pull/60/commits/6869031b618eb9c7709864185fab0ac0a6513534